README.md

```diff
@@ -209,6 +210,20 @@ This testing includes:
 
 This testing is generally done by a tester
 
+### Performance Testing
+
+Performance testing is the practice of evaluating how a system performs in terms of responsiveness and stability under a particular workload. Performance tests are typically executed to examine speed, robustness, reliability of the application.
+
+This testing includes:
+
+- Checking non functional requirements (performance/reliability)
+
+This testing is generally done by a tester
+
+To run the performance tests run the following commands
+
+    make tester-build
+    make performance-test
```

Comment:
Just need some text to explain this target need the developer to be mfa'ed and why.

Reply:
Updated with comment to sign into non-prod